zCore/src/main.rs

```diff
@@ -121,8 +120,7 @@ pub extern "C" fn rust_main(hartid: usize, device_tree_paddr: usize) -> ! {
     });
 
     // 正常由bootloader载入文件系统镜像到内存, 这里不用，而使用后面的virtio
-    let dummy = unsafe { core::slice::from_raw_parts_mut(0 as *mut u8, 0) };
-    main(dummy, boot_info.cmdline);
+    main(&[], boot_info.cmdline);
```

Comment:
Sorry for my fault...
```suggestion
    main(&mut [], boot_info.cmdline);
```